django/chat/llm.py

```diff
@@ -43,6 +47,7 @@ def __init__(
         )
         self._callback_manager = CallbackManager([self._token_counter])
         self.llm = self._get_llm()
+        self.mock_embedding = mock_embedding
```

Comment:
The Mock embedding is for costing or testing purposes. e.g. skipping embedding a bunch of a laws when I'm trying to test how they are loaded. We should add a Mock LLM here later as well, for estimating costs before running expensive processes.